the  current version of sofa-ark-plugin-maven-plugin branch 3.1.x   is  3.1.5,but the file plugin.xml  is 3.0.1-SNAPSHOT

### Describe the question or bug

the  current version of sofa-ark-plugin-maven-plugin branch 3.1.x   is  3.1.5,but the file plugin.xml  is 3.0.1-SNAPSHOT.

### Expected behavior

the file plugin.xml  is 3.1.5.

### Actual behavior

fix the file plugin.xml  3.0.1-SNAPSHOT to  3.1.5.

### Steps to reproduce

Steps to reproduce the problem:
 1. mvn clean compile
 2. See error
the error info :
[ERROR] Invalid plugin descriptor for com.alipay.sofa:sofa-ark-plugin-maven-plugin:3.1.5 (D:\code\outcode\github-sofa-ark\sofa-ark-parent\support\ark-plugin-maven-plugin\target\classes), Plugin's descriptor contains the wrong version: 3.0.1-SNAPSHOT -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/InvalidPluginDescriptorException
### Screenshots

If applicable, add screenshots to help explain your problem.

### Minimal yet complete reproducer code (or GitHub URL to code)
https://github.com/sofastack/sofa-ark/blob/3.1.x/sofa-ark-parent/support/ark-plugin-maven-plugin/src/main/resources/META-INF/maven/plugin.xml

 <version>3.0.1-SNAPSHOT</version>

### Environment

- SOFAArk version: branch 3.1.x
- JVM version (e.g. `java -version`): jdk17
- OS version (e.g. `uname -a`):windows
- Maven version: apache-maven-3.9.8
- IDE version: IntelliJ IDEA 2023.1.3 (Community Edition)

--- a/sofa-ark-parent/support/ark-plugin-maven-plugin/src/main/resources/META-INF/maven/plugin.xml
+++ b/sofa-ark-parent/support/ark-plugin-maven-plugin/src/main/resources/META-INF/maven/plugin.xml
@@ -7,7 +7,7 @@
     <description>A light-weight, java based classloader-isolated framework open-sourced by Ant Financial.</description>
     <groupId>com.alipay.sofa</groupId>
     <artifactId>sofa-ark-plugin-maven-plugin</artifactId>
-    <version>3.0.1-SNAPSHOT</version>
+    <version>3.1.5</version>
     <goalPrefix>sofa-ark</goalPrefix>
     <isolatedRealm>false</isolatedRealm>
     <inheritedByDefault>true</inheritedByDefault>

